src/dodal/beamlines/i22.py

```diff
@@ -108,17 +170,24 @@ def i0(
     return device_instantiation(
         TetrammDetector,
         "i0",
-        "-EA-XBPM-02:",
+        f"-EA-XBPM-0{1 if IS_LAB else 2}:",
```

Comment:
As above, prefered split like saxs

Reply:
here it's just this one bit that is different. if the device is the same physically imo inline ternary just at the point of difference is better